Fixed cache issue

miawgogo opened the pull request:
`{headers: NoCacheHeaders}` needs to be added to the fetchers for the credits

miawgogo committed:
Update global.js

miawgogo committed:
Fix for music

SuperTux88 commented on musicBox.js:
```diff
@@ -1,7 +1,7 @@
 that = this;
 
 function getSong(){
```
what is triggering this `getSong()`? Because if this is also being cached at the moment, and what we see in the server log is only the requests that aren't cached (which would explain the randomness between not working at all, a request every few seconds and 1000 requests a second), then adding the no-cache headers will only end in DOSing himself again. I agree that this request shouldn't be cached, but I think we should find the source of the 1000 requests a second first, to prevent DOSing again. Also, if the overlay is constantly trying to update the song in the background, maybe that could explain some of the frameratey animations because the browser is busy with doing these requests?

If I see it correctly `getSong()` is only called once (at the end of this file), and it schedules a call to itself in a second again. so that should work. But if it initially is called a 1000 times, it also keeps running 1000 times which then means it keeps updating 1000 times a second. So something is weird here. This probably needs more debugging it it really is started multiple times when the overlay loads, or if something else happens.

LoadingArtist replied:
Already discussed on Discord but figured I'd recap here for logging:

I realized why it was updating the song twice a second, it was because the musicBox had a timeline of about a second, and each time it would loop and re-do the musicBox.js code. I've since removed that timeline so it shouldn't be looping/spawning new loops. Now it updates just once a second which is good!

LoadingArtist committed:
Fixed typo

Changed to NoCacheHeaders instead of myHeaders because that's what the var is called
